test/edu/washington/escience/myria/operator/DupElimTest.java

```diff
@@ -94,7 +94,8 @@
   @Test
   public void testDupElim() throws DbException {
     BatchTupleSource src = new BatchTupleSource(makeTestData());
-    StreamingStateWrapper dupElim = new StreamingStateWrapper(src, new DupElim());
+    // StreamingStateWrapper dupElim = new StreamingStateWrapper(src, new DupElim());
```

Comment:
Could you remove the comment? 
(I think it'd be the best if we have another test for the "Stateful" `DupElim` -- I can do it)